src/main/jbake/content/ejb-basicexamples005.adoc

```diff
@@ -304,7 +304,7 @@ Timer timer = timerService.createCalendarTimer(schedule);
 
 For details on the method signatures, see the `TimerService` API
 documentation at
-`http://docs.oracle.com/javaee/7/api/javax/ejb/TimerService.html`.
+`https://javaee.github.io/javaee-spec/javadocs/javax/ejb/TimerService.html`.
```

Comment:
Let me see if I can guess this correctly...

https://jakarta.ee/specifications/platform/8/apidocs/javax/ejb/TimerService.html

Reply:
Indeed you did!